Source/CarthageKit/GitHub.swift

```diff
@@ -167,6 +167,18 @@ extension URLSession {
 		let configuration = URLSessionConfiguration.default
 		configuration.connectionProxyDictionary = Proxy.default.connectionProxyDictionary
 
-		return URLSession(configuration: configuration)
+		let delegate = GitHubURLSessionDelegate()
+		return URLSession(configuration: configuration, delegate: delegate, delegateQueue: OperationQueue.current)
+	}
+}
+
+class GitHubURLSessionDelegate: NSObject, URLSessionTaskDelegate {
```

Comment:
Can you make this `private final`?

Reply:
Made it `internal final` so it'll remain accessible to tests.